python/pubmed_crawler.py

```diff
@@ -48,15 +48,12 @@ def get_args():
         + " and put the results into a CSV file.  Table ID can be provided"
         + " if interested in only scrapping for new publications.")
     parser.add_argument("-g", "--grantview_id",
-                        type=str, default="syn10142562",
+                        type=str, default="syn21918972",
```

Comment:
Just a note that we should eventually change the default to the `grants` table/view in the "CSBC PS-ON DB" project (i.e., not the merged table). However, the table you're using is probably the most up-to-date at the moment.

Reply:
Just want to confirm it is this [table](https://www.synapse.org/#!Synapse:syn21639712/tables/)? There are a couple of `*_grant` tables so I want to ensure I reference the right one.

Reply:
That's the one! Though, it might be worth trying to set up a new 'view' for grants instead (I went with a table earlier because I didn't want to mess up any existing grant annotations. Anyway, another reason not to switch the default yet. :)

Reply:
Sounds good! I just added a TODO note for now :)